.github/pull_request_template.md

```diff
@@ -2,6 +2,6 @@
 <!--- Describe your changes here. --->
 
 ---
-**Target Minecraft Versions:** <!-- 'any' means all supported versions -->
-**Requirements:** <!-- Required plugins, Minecraft versions, server software... -->
```

Comment:
How is `Minecraft version` redundant? I would say `Required Plugins` section is redundant, we barely use that however, it's not really an important chage, we better keep it as is
e.g. Statistics PR #4704
`Any, MC 1.15 (offlineplayers)`

Reply:
You're already asked to provide the target version above it.